packages/server/package.json

```diff
@@ -137,7 +137,7 @@
     "string-similarity": "^3.0.0",
     "stripe": "^4.24.0",
     "tslib": "2.0.3",
-    "uWebSockets.js": "https://github.com/tiffanyhan/uWebSockets.js/tarball/2e733751ce4765c23ca96e3e57265069536ac173"
+    "uWebSockets.js": "https://github.com/adaniels-parabol/uWebSockets.js/tarball/binaries"
```

Comment:
yarn is too aggressive in their caching so if we install the next version from a tarball it'll still use the old version this way.
i think we only used a fork because the version we had was missing the right typings.

i bet we could revert back to master now: `yarn add uWebSockets.js@uNetworking/uWebSockets.js#v19.3.0`